zerver/lib/actions.py

```diff
@@ -1811,10 +1811,7 @@ def check_message(sender: UserProfile, client: Client, addressee: Addressee,
         stream_name = stream_name.strip()
         check_stream_name(stream_name)
 
-        topic_name = addressee.topic()
-        if topic_name is None:
-            raise JsonableError(_("Missing topic"))
-        topic_name = topic_name.strip()
+        topic_name = addressee.topic().strip()
         if topic_name == "":
```

Comment:
Should I move this as well to `for_stream`?

Reply:
@shubhamdhama: I would not refactor this or move anything to `for_stream` at all. It makes sense to have these checks made in `check_message`, which is the main function that validates the contents of a message. I would recommend removing these changes. :)

Reply:
My bad @eeshangarg , I forget to post the link to the conversation, (here is the link)[https://chat.zulip.org/#narrow/stream/backend/subject/action.2Echeck_message/near/456799]  can you please take a look at this, I may have interpreted it wrong.

Reply:
@shubhamdhama: Thanks for the link! I wasn't aware of the rationale behind the change. :) I think it would make sense to move the `if topic_name ==""` check to `for_stream` as well in that case. :)